packages/web/src/components/premium-content-purchase-modal/components/PayToUnlockInfo.tsx

```diff
@@ -30,6 +37,7 @@ export const PayToUnlockInfo = () => {
       <Text className={styles.copy}>
         <span>{messages.copyPart1}</span>
         <Link
+          onClick={handleClickTOSLink}
```

Comment:
it feels like <Link should just have an event to send or something. but that's part of a macro conversation that i'm sure you're dying to have ❤️

Reply:
Haha, you are not wrong. I think Link should always fire an event automatically based on a name/label property. But I think we can/should do that for all clickable/pressable objects automatically :wink: